tools/packaging/kernel/kata_config_version

```diff
@@ -1 +1 @@
-94
+96
```

Comment:
95, but I'd do the bump in the next commit.

Reply:
done